contributing.md

```diff
@@ -1,126 +1,179 @@
+1. After all issues for a particular version have been addressed, the working
+  group votes whether to change thw status of that version and begin the
+  publication process.
+1. If the group does not achieve consensus to change the status, they should
+  identity a specific set of additional issues that must be addressed in order
+  to reach agreement.
+
+### Process Detail
+
+#### Specification Formatting Requirements
+* 80-char fixed-width columns (aka "hard wraps") are required by this
+specification.
+
+#### Specification Versioning 
+The Working Group determines the version number of the next release of the
+specification. Version numbers for the specification indicate the relationship
+of the specification to previous versions according to the guidelines of the
+[Semantic Versioning Specification](https://semver.org). 
+
+#### Issue triage
+Issue triage occurs during spec drafting meetings to determine which version of
+the specification should resolve a particular issue (e.g., the next version or
+some future version). Triage should roughly follow these guiding principles:
+1. Consensus should be easy - this means that if any group participant feels
+  strongly that an issue should be included in the next version of the
+  specification, then it should be included, otherwise it may be included in
+  some future version.
+1. An expectation of doing - this means that any group participant who feels
+  strongly that an issue should be included in the next version of the
+  specification should also be the one to propose concrete changes and work to
+  bring the group to consensus about the changes.
+
+#### Speciation Status and Publishing
+In addition to a version number, the specification has a status:
+1. `Pre-Draft` - The status of the specification is `Pre-Draft` until changed by
+  the Working Group.
+1. `Draft` - This status indicates the specification is ready for feedback from
+  other decentralized identity-related and -adjacent communities. Ideally, these
+  inputs will lead to issues for consensual/deliberative resolution. Further
+  validation and improvement of the specification should be done via
+  implementation and testing. Once the Working Group approves a version of the
+  specification as a `Draft Deliverable` (i.e. it has been given `Draft`
+  status), the `Draft Deliverable` becomes the basis for all work going forward
+  on that version (in accordance with the DIF Charter Deliverable Development
+  Process).
+1. `Working Group Approved` - Once the Working Group has refined the `Draft
+  Deliverable` and believes it has addressed all pertinent issues, it may
+  progress the `Draft Deliverable` to `Working Group Approved` status (in
+  accordance with the DIF Charter Deliverable Development Process). 
+  
+A specification with the `Working Group Approved` status indicates that all of
+the following conditions are met: 
+   * There is quality code that demonstrates the concepts it describes.
```

Comment:
Might be good to specify test vectors here as well, while code it nice, test vectors encourage implementations in other languages, which will further support the standard. 

The first implementation should be used to generate the test vectors, using as much determinism as possible.

Reply:
Something like this?
```suggestion
   * There is quality code that demonstrates the concepts it describes.
   * There are test vectors which may be used to test implementations.
```

Reply:
Yep, I would approve with that change.

Reply:
@OR13 can you approve?